src/Import/BuiltIn/TsImportCollector.php

```diff
@@ -18,19 +18,24 @@
 {
```

Comment:
Shouldn't this be called a `Es6ImportCollector` then?

Reply:
that is the question yes